dawn/src/dawn/Validator/WeightChecker.cpp

```diff
@@ -46,16 +46,6 @@ void WeightChecker::WeightCheckerImpl::visit(
                         .isDense();
   }
 }
-void WeightChecker::WeightCheckerImpl::visit(const std::shared_ptr<dawn::ast::FunCallExpr>& expr) {
-  if(parentIsWeight_) {
-    weightsValid_ = false;
-    return;
-  } else {
-    for(const auto& s : expr->getChildren()) {
-      s->accept(*this);
```

Comment:
We don't need this, because the default implementation will do this anyways?

Reply:
correct!